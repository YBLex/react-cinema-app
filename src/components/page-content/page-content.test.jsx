import React from 'react';
import renderer from 'react-test-renderer';

import PageContent from './page-content.jsx';

const Mock = () => <div className="mock"></div>;

it(`Render PageContent`, () => {
  const tree = renderer.create(
      <PageContent>
        <Mock />
      </PageContent>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
