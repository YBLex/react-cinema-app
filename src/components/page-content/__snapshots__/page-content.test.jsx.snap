// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageContent 1`] = `
<div
  className="page-content"
>
  <div
    className="mock"
  />
</div>
`;
